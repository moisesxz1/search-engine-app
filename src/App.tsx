import React from 'react';
import SearchEngineContainer from './pages/SearchEngine/SearchEngineContainer/SearchEngineContainer';

function App() {
  return <SearchEngineContainer />;
}

export default App;
