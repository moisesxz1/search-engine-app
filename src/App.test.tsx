import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import App from './App';

describe('Tests on the App />Component', () => {
  test('It should render <App /> correctly', () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
