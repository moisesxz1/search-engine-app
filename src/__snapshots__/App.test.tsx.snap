// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on the App />Component It should render <App /> correctly 1`] = `<SearchEngineContainer />`;
